fahrschein-http-api/src/main/java/org/zalando/fahrschein/http/api/ContentEncoding.java

```diff
@@ -1,15 +1,42 @@
 package org.zalando.fahrschein.http.api;
 
+import com.github.luben.zstd.ZstdOutputStream;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.zip.GZIPOutputStream;
+
 public enum ContentEncoding {
-    IDENTITY("identity"), GZIP("gzip");
+    IDENTITY("identity"), GZIP("gzip"), ZSTD("zstd");
```

Comment:
It's really elegant how adding a new `ContentEncoding` immediately makes it part of the `NakadiClientEnd2EndTest` for every existing http implementation. :smile: